frontend/public/components/nav/_perspective-nav.scss

```diff
@@ -0,0 +1,25 @@
+.oc-nav-pinned-item {
+  display: flex;
+  align-items: center;
+  padding-right: var(--pf-global--spacer--sm);
+  .pf-c-nav__item {
+    flex: 1;
+  }
+  .pf-c-nav__link {
+    padding-bottom: 0;
+    padding-left: 0;
+    padding-top: 0;
+  }
+  .oc-nav-pinned-item__unpin-button {
+    visibility: hidden;
+    &:focus {
+      visibility: visible;
+    }
+  }
+  &:hover, &:focus {
+    .oc-nav-pinned-item__unpin-button {
+      visibility: visible;
```

Comment:
Use opacity instead of visibility so that the node can be navigated to via keyboard tab.